content/tboerger/drone-ansible/index.md

```diff
@@ -0,0 +1,84 @@
+---
+version: '1.0'
+date: 2019-02-15T00:00:00+00:00
+title: Ansible
+author: tboerger
```

Comment:
it is indeed a bit mixed up in the repo. there are individual's repos with their plugins and then there's the drone-plugins folder. would you suggest i move everthing below that folder as well?

Reply:
Only move the folder drone-ansible to drone-plugins. Yes it is mixed because some plugins are a bit more "official" and managed by the drone-plugins organization and other plugins are managed by individual users and those will have their own folder.

Reply:
i see and understand. so because @tboerger is part of the org it's considered 'official'? then i will move it there.